packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/tasks/tasks.html

```diff
@@ -0,0 +1,96 @@
+<oui-back-button
+    data-previous-page="{{:: 'pci_databases_connectors_tasks_back' | translate }}"
+    data-on-click="$ctrl.goBack()"
+></oui-back-button>
+
+<div class="d-flex justify-content-between align-items-center mb-4">
+    <div>
+        <h2
+            class="mb-0"
+            data-translate="pci_databases_connectors_tasks_title"
+            data-translate-values="{name: $ctrl.connector.name}"
+        ></h2>
+        <span class="oui-paragraph">
+            <span>Type: {{$ctrl.connector.connectorInformation.name}}</span>
```

Comment:
Nitpick: it might be better idea to generate this static content in a controller. Its make code easy to read.

Reply:
Same as here: https://github.com/ovh/manager/pull/6577#discussion_r822545955, I'm not sure how to handle the link ?